specs/flipper/flipper.md

```diff
@@ -0,0 +1,105 @@
+# Flipper
+
+## Overview
+
+The flipper component is most often used to page through blocks of content or collections of ui elements.
+
+### Use Cases
+- Tony is looking for a new refrigerator. He goes onto his computer and navigates the browser to a site that sells appliances. He sees a list of refrigerators but not the one he wants so he clicks an arrow button (flipper) to show more. He sees a new GE Cafe that interests him and clicks on it which directs him to a purchase page.
+
+- Bobby is blind and is also looking for a new refrigerator. He goes onto his computer and navigates the browser to the same site that sells appliances. He tabs through a list of refrigerators until he finds the one that was recommended to him by a friend. Bobby is unaware of the flipper.
+
+### Non-goals
+- Other button implementations
+
+### Prior Art/Examples
+Limited prior art is available from major component libraries, however many brands and products leverage flippers on the front page of their sites:
+- Hulu
+- Netflix
+- Amazon
+- Microsoft
+
+---
+
+
+### API
+
+Extends FAST Element
+
+*Component Name*
+- `fast-flipper`
+
+*Attrs*
+- `hiddenFromAT` - The control is hidden from assistive technology. Defaults to true
```

Comment:
Feedback is welcome on a better name here :). Per previous comments by @EisenbergEffect, something like `hiddenFromAssistiveTechnologies` is a bit verbose. As an FYI, this convention was used here as an initial stub because the same language is often used by the W3C spec to denote when something is not exposed to AT users.

Reply:
We could maybe do something like: 

```
@attr({attribute: "aria-hidden"})
public hiddenFromAT: boolean = true;
```

Then then user could simply write `<fast-flipper aria-hidden="false" />` to override. Note that this doesn't seem to actually work - aria-hidden is overridden by the default property value but I think that might be a bug. You might also need to do write a custom converter.

Reply:
Hmm. Overwriting built-ins may not be something we can support. I'm not sure. It would be nice to use aria-hidden if we can. Do we need to do anything in our own code in response to this changing?

Reply:
I think it just needs to be hidden *by default* - perhaps when connected we could just check to see if the attribute exists, and if not set it imperatively.

Reply:
That's an interesting idea. What do you think @chrisdholt ?

Reply:
So, if `aria-hidden` exists on the host, use that value and respond accordingly. If it doesn't exist, set aria-hidden in connectedCallback?

Reply:
@nicholasrice @EisenbergEffect just want to confirm I'm understanding the proposed implementation (above)

Reply:
Confirmed offline, I like this idea and think we should pursue it in implementation. 👍